abc.templates/base-workflows/contents/workflows/reusable-guardian-plan.yml

```diff
@@ -47,8 +40,6 @@ jobs:
     # We do not run any plan jobs in the merge queue. But these jobs are still
     # required to run (in a skipped state) in order to block adding to the
     # merge queue until a plan has been created.
-    if: |
```

Comment:
i think wee need to keep this but make it pull_request, this was added to support the mrege_group event i believe